doc/index.md

```diff
@@ -55,4 +55,5 @@ minversions.md
 team.md
 contributing.md
 maintenance.md
+ecosystem.md
```

Comment:
I am wondering if it makes sense to have `ecosystem.md` in one section together with `minversions.md`. Currently `minversions.md` is listed under "Reference documentation" and `ecosystem.md` under "Development".

Reply:
Makes sense. Done in 5bf8c5dbeeb62cf6e207a7743bfdbe19bef2628d.